### PR TITLE
Markdown fixes for imported CurriculumBuilder content

### PR DESCRIPTION
**Pull request by Hamms**
Using some one-off scripts (captured in https://github.com/code-dot-org/code-dot-org/pull/38631 and https://github.com/code-dot-org/curriculumbuilder/pull/336) and an HTML comparison tool (https://github.com/Hamms/html-equivalency), I identified all the pieces of content which have rendering differences between CurriculumBuilder and Code Studio.

This PR represents the subset of those differences that result from minor formatting differences.

## Future Work

Next up, I'll be working on the differences that are the result of the various different special syntaxes we have in CB.

## Testing story

All these content changes were done in the levelbuilder editor and were visually inspected for correctness.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/scripts_json/csd3-2021.script_json [hunk 1]
```diff
@@ -5090,7 +5090,7 @@
       "key": "b2ad1d43-99f9-40fa-9f88-d8b42f8c723e",
       "position": 7,
       "properties": {
-        "description": "Using the rubric, students should assess their own project before submitting it.\r\n\r\n\r\n&fa-desktop; Send students to Code Studio to complete their reflection on their attitudes toward computer science. Although their answers are anonymous, the aggregated data will be available to you once at least five students have completed the survey.",
```
Comment (bethanyaconnor) [inline dashboard/config/scripts_json/csd3-2021.script_json:5093]: I don't know if we ever came to a consensus on whether to support `&fa-desktop`

Comment (Hamms) [reply dashboard/config/scripts_json/csd3-2021.script_json:5093]: In this specific case, it looked like it was being used to display the "there's a tip" icon, which of course with the new system is automatically showing up when a "tip" is present.

I did also notice these fontawesome icons in a couple of other places and left them untouched, assuming that we'd get to them as part of the larger "special CB syntaxes" effort

Comment (bethanyaconnor) [reply dashboard/config/scripts_json/csd3-2021.script_json:5093]: I think they were used to indicate that students should move to the computer at that point

File: dashboard/config/scripts_json/csp1-2021.script_json [hunk 2]
```diff
@@ -4032,22 +4032,11 @@
...
-        "lesson_activity.key": "7c9d4a66-304f-418f-977e-a986e31cb20e"
-      }
-    },
     {
       "key": "eefa3b88-16f2-48f7-8b9a-c37dcadc92bd",
-      "position": 8,
+      "position": 7,
       "properties": {
-        "description": "[slide]  Video #1: [Copyright Overview](https://www.youtube.com/watch?v=GPNWvU_IphU) (2 minutes)\r\n* [slide] Video #2: [Copyright in Practice](https://www.youtube.com/watch?v=ITASCSxnBqw) (4 minutes)",
+        "description": "* [slide]  Video #1: [Copyright Overview](https://www.youtube.com/watch?v=GPNWvU_IphU) (2 minutes)\n* [slide] Video #2: [Copyright in Practice](https://www.youtube.com/watch?v=ITASCSxnBqw) (4 minutes)\n* [slide] Video #3: [Creative Commons Copyright](https://www.youtube.com/watch?v=wC-m23T7cPM) (3 minutes)",
```
Comment (bethanyaconnor) [inline dashboard/config/scripts_json/csp1-2021.script_json:4039]: Thanks for cleaning this up! A known bug in the import :(

File: dashboard/config/scripts_json/coursec-2021.script_json [hunk 3]
```diff
@@ -1914,7 +1914,7 @@
...
... As the teacher, let them know that you are SO SMART that you already put all of the code in that you are going to need to solve this puzzle, then ask them to watch you \"Run\" it by moving your finger (or a penny, or some other indicator) along the path.\n\nIt won't be long before you run into a block of TNT.  Feign frustration.\n\n**Discuss:**\n\n- What am I feeling right now, do you think?\n- Should I quit?\n- Should I throw all of the code away and start over?\n\n**Think:** How can I fix this program so that I don't run into the TNT?\n\n**Pair:** Have students work on solutions to get the bird around the TNT. Depending on your classroom, you might want to either have them fix each mistake one at a time (with demos in between) or students might feel comfortable working together to fix the entire program.\n\n**Share:** Have volunteers come up to help move the blocks into the right location.  \"Run\" the program over and over as a class, fixing bugs, until the bird does what it is supposed to.  Continue to point out experiences that relate to persistence, frustration, and debugging.\n\nWhen your class reaches the pig, celebrate not only their achievements, but their persistence!",
```
Comment (davidsbailey) [inline dashboard/config/scripts_json/coursec-2021.script_json:1917]: I had to use a different tool to see what changed on this line:

Screen Shot 2021-01-21 at 4 20 08 PM

are there next steps that need to be tracked for this image, whose width is being removed?

**Review by davidsbailey — APPROVED** [review]
Nice job getting up to speed and getting these edits into the script_json, @Hamms ! One question inline.

File: dashboard/config/scripts_json/coursec-2021.script_json (hunk 3 above)
Comment (Hamms) [reply dashboard/config/scripts_json/coursec-2021.script_json:1917]: nope; in this case, the image being used never needed to be a size other than its display size, so I just resized the actual image. See the commit description https://github.com/code-dot-org/code-dot-org/pull/38669/commits/79103952dc26fc2e6bd702418a1e0aae8c518a1e

File: dashboard/config/scripts_json/csd3-2021.script_json (hunk 1 above)
Comment (dmcavoy) [reply dashboard/config/scripts_json/csd3-2021.script_json:5093]: Ya fa-desktop was used to mark where there was a transition onto the computer. I think when we talked to the curriculum team they didn't seem attached to it

Comment (dmcavoy) [reply dashboard/config/scripts_json/csd3-2021.script_json:5093]: I think if we didn't remove all the fa-desktops we should keep these ones and double check with curriculum before removing them all at once.

Comment (Hamms) [reply dashboard/config/scripts_json/csd3-2021.script_json:5093]: ahhh, I definitely misunderstood what this one was being used for, then. I'll put it back for now

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**